apps/crn-server/test/integration/squidex/reminders.integration-test.ts

```diff
@@ -517,6 +519,314 @@ describe('Reminders', () => {
+        documentType: researchOutputInput.documentType,
+        sharingStatus: researchOutputInput.sharingStatus,
+        title: researchOutputInput.title,
+        authors: researchOutputInput.authors,
+        environmentIds: researchOutputInput.environmentIds,
+        labIds: researchOutputInput.labIds,
+        methodIds: researchOutputInput.methodIds,
+        organismIds: researchOutputInput.organismIds,
+        keywordIds: researchOutputInput.keywordIds,
+        teamIds: researchOutputInput.teamIds,
+        updatedBy: creatorId,
+        workingGroups: researchOutputInput.workingGroups || [],
+      });
+
+      await retryable(async () => {
+        const researchOutput = await researchOutputDataProvider.fetchById(
+          researchOutputId,
+        );
+
+        const expectedReminder: ResearchOutputInReviewReminder = {
+          id: `research-output-in-review-${researchOutputId}`,
+          entity: 'Research Output',
+          type: 'In Review',
+          data: {
+            researchOutputId,
+            associationName: workingGroup.data.title.iv,
+            associationType: 'working group',
+            title: researchOutputInput.title,
+            addedDate: researchOutput?.created.slice(0, -5) + 'Z' || '',
+            documentType: researchOutputInput.documentType,
+            reviewRequestedBy: creatorUsername,
+          },
+        };
+        const reminders = await reminderDataProvider.fetch(
+          fetchRemindersOptions,
+        );
+
+        expect(reminders).toEqual({
+          total: 1,
+          items: [expectedReminder],
+        });
+      });
+    });
+
+    test('Should not see the reminder when the user is not PM within the working group that owns the RO', async () => {
+      const wgData = createWorkingGroup(creatorId);
+
+      const workingGroup = await workingGroupRestClient.create(
+        wgData as any,
+        true,
+      );
+      const workingGroupId = workingGroup.id;
+
+      const researchOutputInput = getResearchOutputInputDraft(
+        teamId,
+        creatorId,
+        workingGroupId,
+      );
+
+      const researchOutputId = await researchOutputDataProvider.create(
+        researchOutputInput,
+        {
+          publish: false,
+        },
+      );
+
+      await researchOutputDataProvider.update(researchOutputId, {
+        reviewRequestedById: creatorId,
+        documentType: researchOutputInput.documentType,
+        sharingStatus: researchOutputInput.sharingStatus,
+        title: researchOutputInput.title,
+        authors: researchOutputInput.authors,
+        environmentIds: researchOutputInput.environmentIds,
+        labIds: researchOutputInput.labIds,
+        methodIds: researchOutputInput.methodIds,
+        organismIds: researchOutputInput.organismIds,
+        keywordIds: researchOutputInput.keywordIds,
+        teamIds: researchOutputInput.teamIds,
+        updatedBy: creatorId,
+        workingGroups: researchOutputInput.workingGroups || [],
```

Comment:
This could be done in this test and the other ones that do this update 😉 
```suggestion
       ...researchOutputInput,
        reviewRequestedById: creatorId,
        updatedBy: creatorId,
        workingGroups: researchOutputInput.workingGroups || [],
```